libs/base/core.cpp

```diff
@@ -561,49 +561,50 @@ int length(String s) {
 }
 
 #define isspace(c) ((c) == ' ')
+#define iswhitespace(c) ((c) == 0x0009 || (c) == 0x000B || (c) == 0x000C || (c) == 0x0020 || (c) == 0x00A0 || (c) == 0xFEFF || (c) == 0x000A || (c) == 0x000D || (c) == 0x2028 || (c) == 0x2029)
```

Comment:
c is 8 bit, so characters above 0xff won't ever match; if you want to support unicode it's a bit more tricky

Reply:
ahh forgot about that, so that's line separator, paragraph separator and zero width non breaking space; I doubt anyone will end up using those ever with parsefloat so will just drop for now